device/core/devdoc/device_client_requirements.md

```diff
@@ -265,6 +273,20 @@ interface DeviceMethodEventHandler {
 
 **SRS_NODE_DEVICE_CLIENT_16_066: [** The client shall emit an error if connecting the transport fails while subscribing to message events **]**
 
+#### inputMessage
```

Comment:
>inputMessage [](start = 5, length = 12)

I'm not sure how I feel about this:
1. if we have a specific property, why not use the regular message event?
2. if we have a specific property, why not use the regular sendEvent method?